cmd/argocd/commands/app.go

```diff
@@ -759,7 +733,7 @@ func printAppResources(w io.Writer, app *argoappv1.Application, showOperation bo
 }
 
 // printAppStateChange prints a component state change if it was different from the last time we saw it
-func printAppStateChange(w io.Writer, prevComp *argoappv1.ComparisonResult, app *argoappv1.Application) {
+func printAppStateChange(w io.Writer, prevComp *argoappv1.ComparisonResult, prevState *argoappv1.OperationState, app *argoappv1.Application) {
```

Comment:
prevState is passed but not used anywhere? How are you comparing previous operation state information with the current?

Reply:
@jessesuen good catch!